django/contrib/csp/utils.py

```diff
@@ -0,0 +1,48 @@
+from django.conf import settings
+
+
+def from_settings():
+    return {
+        'default-src': getattr(settings, 'CSP_DEFAULT_SRC', ["'self'"]),
+        'script-src': getattr(settings, 'CSP_SCRIPT_SRC', None),
+        'object-src': getattr(settings, 'CSP_OBJECT_SRC', None),
+        'style-src': getattr(settings, 'CSP_STYLE_SRC', None),
+        'img-src': getattr(settings, 'CSP_IMG_SRC', None),
+        'media-src': getattr(settings, 'CSP_MEDIA_SRC', None),
+        'frame-src': getattr(settings, 'CSP_FRAME_SRC', None),
+        'font-src': getattr(settings, 'CSP_FONT_SRC', None),
+        'connect-src': getattr(settings, 'CSP_CONNECT_SRC', None),
+        'sandbox': getattr(settings, 'CSP_SANDBOX', None),
+        'report-uri': getattr(settings, 'CSP_REPORT_URI', None),
+    }
```

Comment:
The fact that you're immediately wrapping all `CSP_*` settings into a dict suggests that it should have been a dict in the first place.